process/project-lifecycle.md

```diff
@@ -0,0 +1,161 @@
+
+<!-- TOC -->
+
+  - [Stages - Definitions & Expectations](#stages---definitions--expectations)
+      - [Sandbox](#sandbox)
+          - [Gives - Review and Reporting](#gives---review-and-reporting)
+          - [Gets - Benefits](#gets---benefits)
+          - [Sandbox Entry Requirements](#sandbox-entry-requirements)
+          - [Project Graduation Process: Sandbox to Incubating](#project-graduation-process-sandbox-to-incubating)
+          - [Submission Forms for Sandbox](#submission-forms-for-sandbox)
+          - [Submission Forms for Sandbox to Incubating](#submission-forms-for-sandbox-to-incubating)
+      - [Incubating](#incubating)
+          - [Gives - Review and Reporting](#gives---review-and-reporting)
+          - [Gets - Benefits](#gets---benefits)
+          - [Incubating Entry Requirements](#incubating-entry-requirements)
+          - [Project Graduation Process: Incubating to Graduation](#project-graduation-process-incubating-to-graduation)
+          - [Submission Forms for Incubating](#submission-forms-for-incubating)
+          - [Submission Forms for Incubating to Graduation](#submission-forms-for-incubating-to-graduation)
+      - [Graduated](#graduated)
+          - [Gives - Review and Reporting](#gives---review-and-reporting)
+          - [Gets - Benefits](#gets---benefits)
+          - [Project Graduation Process: Graduated to Archiving](#project-graduation-process-graduated-to-archiving)
+          - [Submission Forms for Graduated to Archived](#submission-forms-for-graduated-to-archived)
+      - [Archived](#archived)
+
+<!-- /TOC -->
+
+
+## Stages - Definitions & Expectations
+
+Projects have a maturity level of sandbox, incubating, or graduated. Archived is for Projects no longer in active development. The maturity level is a signal by OpenSSF as to what sorts of enterprises should be adopting different projects. Projects increase their maturity by demonstrating their sustainability to OpenSSF’s Technical Advisory Committee(TAC): that they have adoption, a healthy rate of changes, committers from multiple organizations, have adopted the OpenSSF Code of Conduct, and have achieved and maintained the Core Infrastructure Initiative Best Practices Badge.
+
+![Project Stages](/process/project-stages.png)
+
+### Sandbox
+
+The OpenSSF Sandbox is the entry point for early stage Projects and has four goals:
+
+* Encourage public visibility of experiments or other early work that can add value to the OpenSSF mission and build the ingredients of a successful Incubation level Project
+* Facilitate alignment with existing Working Groups or Projects if (and only if) this is desired
+* Nurture Projects (e.g. via OpenSSF Service Desk requests)
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to OpenSSF legal, code of conduct, and IP Policy requirements.
+
+#### Gives - Review and Reporting
+*** Need Text ***
+
+#### Gets - Benefits
+*** Need Text ***
```

Comment:
I think OpenSSF might do well to borrow the ASF approach for [branding of podling vs incubators](https://incubator.apache.org/guides/branding.html). Because of the momentum behind open source security at the moment, I think reserving some marketing benefits until projects reach an incubation or graduation stage would help keep the focus on developing the technology of the Sandbox projects and encourages projects to progress.

Reply:
Agreed.

Reply:
I'm leaving this to be addressed by PR#100.